app/views/cdo/create/owner-summary.njk

```diff
@@ -0,0 +1,88 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% extends '_layout.njk' %}
+
+{% block pageTitle %}{{ serviceName }}{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m">Confirm details</h2>
+
+    <form method="post" action="{{ model.formAction }}">
+      {{ govukSummaryList({
```

Comment:
I think the GDS Summary List is one of the components where it makes sense more sense to use HTML rather than nunjucks.

I personally feel the HTML version is a lot more readable. If it saves you time, you can just use the HTML from the prototype.